code/game/objects/machinery/sentries.dm

```diff
@@ -1265,7 +1265,7 @@
 	icon_state = "minisentry_packed"
 	item_state = "minisentry_packed"
 	w_class = WEIGHT_CLASS_BULKY
-	max_integrity = 155 //We keep track of this when folding up the sentry.
+	max_integrity = 200 //We keep track of this when folding up the sentry.
```

Comment:
Instead of just increasing its health you should correctly set its `obj_integrity` to 155.